Assets/Mirror/Examples/MultipleAdditiveScenes/Scripts/PhysicsCollision.cs

```diff
@@ -0,0 +1,44 @@
+﻿using UnityEngine;
+
+namespace Mirror.Examples.MultipleAdditiveScenes
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public class PhysicsCollision : NetworkBehaviour
+    {
+        [Tooltip("how forcefully to push this object")]
+        public float force = 12;
+
+        public Rigidbody rigidbody3D;
+
+        private void OnValidate()
+        {
+            if (rigidbody3D == null)
+                rigidbody3D = GetComponent<Rigidbody>();
+        }
+
+        private void Start()
+        {
+            rigidbody3D.isKinematic = !isServer;
+        }
+
+        [ServerCallback]
+        void OnCollisionStay(Collision other)
+        {
+            if (other.gameObject.CompareTag("Player"))
+            {
+                // get direction from which player is contacting object
+                Vector3 dir = other.contacts[0].normal;
```

Comment:
you could name it "direction" for consistency with the other "direction" variable in the other script.
also, this way it's more obvious what it is :)

Reply:
Done